groupy/client.py

```diff
@@ -61,6 +62,7 @@ def _try_fetch(self, path, **kwargs):
             try:
                 return self._fetch(path, **kwargs)
             except exc.BackendConnectionError as err:
+                logging.warning("Marking server {} as dead.".format(err.server))
```

Comment:
This will blow up unless we also bump clowncar, which we really should, but let's also replace this with `err.server.hostname` or similar to not leak clowncar stuff.